force-app/main/default/lwc/registrationForm/registrationForm.js

```diff
@@ -0,0 +1,94 @@
+import { LightningElement } from 'lwc';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import { refreshApex } from '@salesforce/apex';
+import isEmailRegistered from '@salesforce/apex/MeetupRegistrationService.isEmailRegistered';
+import getRegisterableMeetup from '@salesforce/apex/MeetupService.getRegisterableMeetup';
+
+export default class RegistrationForm extends LightningElement {
+    validMeetup = true;
+    registrationAllowed = true;
+    showForm = false;
+    meetupName;
+    meetupId;
+
+    connectedCallback() {
+        // Prepare the initial state of the page, lookup up the meetup and validating it's registerability.
+        const urlParamCode = new URLSearchParams(window.location.search).get('c__code');
```

Comment:
The original reqs state "RegistrationPage?code=12345" but what I've found is that the platform expects query params to be namespaced in the custom/local namespace with c__ like this.